packages/core/src/components/Spinner/Spinner.scss

```diff
@@ -113,11 +113,22 @@ Style guide: components.spinner
 
 - If the process takes a long time, use something else.
 
+## Accessibility Considerations
```

Comment:
For consistency with [other pages](https://design.cms.gov/components/tabs/#guidance), let's change this to "Accessibility"